gnome-doc-utils.make is horribly parallel unsafe, as reported on several
distro bugzillas (see Gentoo bugs 252163, 255479, 260611, 260827).
I have yet to find the root cause or a proper solution, but in the meantime
we can force g-d-u help dirs to run serially without losing parallel make
for the rest of the package:

http://www.gnu.org/software/make/manual/html_node/Special-Targets.html

--- origsrc/gnome-doc-utils-0.16.1/tools/gnome-doc-utils.make	2008-10-27 19:28:14.000000000 -0500
+++ src/gnome-doc-utils-0.16.1/tools/gnome-doc-utils.make	2009-06-08 15:30:24.095006600 -0500
@@ -576,3 +576,5 @@ uninstall-doc-omf:
 	  echo "rm -f $(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)/$$omf"; \
 	  rm -f "$(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)/$$omf"; \
 	done
+
+.NOTPARALLEL:
